src/operators/localoperator.jl

```diff
@@ -178,3 +190,17 @@ function costfun(peps::InfinitePEPS, env, op::NLocalOperator{NearestNeighbor})
     ov = sum(expectation_value(rotl90(peps), rotl90(env), op))
     return real(oh + ov)
 end
+
+"""
+    costfun(peps::InfinitePEPS, env, op::AnisotropicNNOperator)
+    
+Compute the expectation value of an on-site and an anisotropic nearest-neighbor operator.
+This is used to evaluate and differentiate the energy in ground-state PEPS optimizations.
+"""
+function costfun(peps::InfinitePEPS, env, op::AnisotropicNNOperator)
+    oos = expectation_value(peps, env, op.h0)[1]
```

Comment:
Why is this one not a `sum`? Seems like it should be.

Reply:
I agree